src/Employer/Employer.Web/Configuration/ManageApprenticeshipsLinkHelper.cs

```diff
@@ -15,6 +15,7 @@ public ManageApprenticeshipsLinkHelper(IOptions<ExternalLinksConfiguration> exte
         }
 
         public string MaRoot => _externalLinks.ManageApprenticeshipSiteUrl;
+        public string AccountHome => $"{_externalLinks.ManageApprenticeshipSiteUrl}{_maRoutes.ManageApprenticeshipSiteAccountsHome}";
```

Comment:
`_maRoutes.ManageApprenticeshipSiteAccountsHomeRoute` because that is how the other properties on the `ManageApprenticeshipsRoutes` class/config are named.

Reply:
👍